README.md

```diff
@@ -84,6 +84,29 @@ $ curl -X GET http://localhost:8008/v1
 into, run `nexus` with `CONSENSUS_DAMASK_GENESIS_DUMP=/tmp/genesis.sql`. The SQL will be written to the indicated
 file, provided that genesis hasn't been parsed into the DB yet. The easiest way to achieve the latter is to wipe the DB.
 
+### Code quality tools
+
+Here are our recommendations for getting the tools that `make lint` and `make fmt` use.
+
+**golangci-lint**: [use their `curl | sh` installer](https://golangci-lint.run/usage/install/).
+They explain that the tool is in dependency hell and any more structured distribution of it might not work.
+
+```sh
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
+```
+
+**gofumpt**: [go install](https://pkg.go.dev/mvdan.cc/gofumpt)
+
+```sh
+go install mvdan.cc/gofumpt@latest
+```
+
+**goimports**: [go installl](https://pkg.go.dev/golang.org/x/tools/cmd/goimports)
+
+```sh
+go install golang.org/x/tools/cmd/goimports@latest
```

Comment:
`latest` always makes me nervous. These are just manual instructions, so I guess it's fine ... Your call.
FWIW, I have
```
$ go list -m -json golang.org/x/tools  # contains goimports
{
        "Path": "golang.org/x/tools",
        "Version": "v0.11.0",
        "Time": "2023-07-05T15:50:13Z",
        "Indirect": true
}

$ gofumpt -version
v0.4.0 (go1.19.1)
```
and CI uses [golangci-lint v1.49](https://github.com/oasisprotocol/nexus/blob/2e858695add9655b1b4bedeb67c0aee0bb64a815/.github/workflows/ci-lint.yaml#L51-L51)

Reply:
ideally this and gofumpt similarly should use the same version that golangci-lint uses. but bleh

Reply:
I'll leave the instruction at latest but add a note